Mu2eG4/geom/geom_common_no_tsu_ps.txt

```diff
@@ -4,7 +4,7 @@
 // The DS field map is removed to save space
```

Comment:
This should say the TS upstream and PS fields are removed, and the DS is not hidden

Reply:
Thanks.  Fixed and pushed.